dashboard/client/src/api/functionsApi.test.js

```diff
@@ -70,4 +71,20 @@ describe('functionsApi', () => {
       expect(second.name).toEqual(functionResponse1.name);
     });
   });
+
+
+  describe('replaceFunctionEnding', () => {
```

Comment:
Please add a case where it does nothing, for when it's not appropriate - i.e. `function` is not in the sub-domain.